pkg/theia/commands/policy_recommendation_run_test.go

```diff
@@ -143,3 +156,355 @@ func TestPolicyRecommendationRun(t *testing.T) {
+			cmd.Flags().String("end-time", "2006-01-03 15:04:05", "")
+			cmd.Flags().String("ns-allow-list", "[\"kube-system\",\"flow-aggregator\",\"flow-visibility\"]", "")
+			cmd.Flags().Bool("exclude-labels", true, "")
+			cmd.Flags().Bool("to-services", true, "")
+			cmd.Flags().Int32("executor-instances", 1, "")
+			cmd.Flags().String("driver-core-request", "1", "")
+			cmd.Flags().String("driver-memory", "1m", "")
+			cmd.Flags().String("executor-core-request", "1", "")
+			cmd.Flags().String("executor-memory", "mock_executor-memory", "")
+		case "Unspecified file":
+			cmd.Flags().String("type", "initial", "")
+			cmd.Flags().Int("limit", 0, "")
+			cmd.Flags().String("policy-type", "anp-deny-applied", "")
+			cmd.Flags().String("start-time", "2006-01-02 15:04:05", "")
+			cmd.Flags().String("end-time", "2006-01-03 15:04:05", "")
+			cmd.Flags().String("ns-allow-list", "[\"kube-system\",\"flow-aggregator\",\"flow-visibility\"]", "")
+			cmd.Flags().Bool("exclude-labels", true, "")
+			cmd.Flags().Bool("to-services", true, "")
+			cmd.Flags().Int32("executor-instances", 1, "")
+			cmd.Flags().String("driver-core-request", "1", "")
+			cmd.Flags().String("driver-memory", "1m", "")
+			cmd.Flags().String("executor-core-request", "1", "")
+			cmd.Flags().String("executor-memory", "1m", "")
+		case "Unspecified use-cluster-ip":
+			cmd.Flags().String("type", "initial", "")
+			cmd.Flags().Int("limit", 0, "")
+			cmd.Flags().String("policy-type", "anp-deny-applied", "")
+			cmd.Flags().String("start-time", "2006-01-02 15:04:05", "")
+			cmd.Flags().String("end-time", "2006-01-03 15:04:05", "")
+			cmd.Flags().String("ns-allow-list", "[\"kube-system\",\"flow-aggregator\",\"flow-visibility\"]", "")
+			cmd.Flags().Bool("exclude-labels", true, "")
+			cmd.Flags().Bool("to-services", true, "")
+			cmd.Flags().Int32("executor-instances", 1, "")
+			cmd.Flags().String("driver-core-request", "1", "")
+			cmd.Flags().String("driver-memory", "1m", "")
+			cmd.Flags().String("executor-core-request", "1", "")
+			cmd.Flags().String("executor-memory", "1m", "")
+			cmd.Flags().String("file", "filename", "")
+		case "Unspecified waitFlag":
+			cmd.Flags().String("type", "initial", "")
+			cmd.Flags().Int("limit", 0, "")
+			cmd.Flags().String("policy-type", "anp-deny-applied", "")
+			cmd.Flags().String("start-time", "2006-01-02 15:04:05", "")
+			cmd.Flags().String("end-time", "2006-01-03 15:04:05", "")
+			cmd.Flags().String("ns-allow-list", "[\"kube-system\",\"flow-aggregator\",\"flow-visibility\"]", "")
+			cmd.Flags().Bool("exclude-labels", true, "")
+			cmd.Flags().Bool("to-services", true, "")
+			cmd.Flags().Int32("executor-instances", 1, "")
+			cmd.Flags().String("driver-core-request", "1", "")
+			cmd.Flags().String("driver-memory", "1m", "")
+			cmd.Flags().String("executor-core-request", "1", "")
+			cmd.Flags().String("executor-memory", "1m", "")
+			cmd.Flags().String("file", "filename", "")
+			cmd.Flags().Bool("use-cluster-ip", true, "")
```

Comment:
A Similiar case here, consider my suggestion to deduplicate code from line 277 to 499.

Reply:
can find an approach for now, resolving the conversation